Directory.Build.targets

```diff
@@ -22,6 +22,15 @@
   </PropertyGroup>
 
   <!-- Packaging -->
+  <ItemGroup Condition="'$(IsPackable)' == 'true'">
+    <None Include="$(LicenseFile)"
```

Comment:
Why `None` and not `File` with `SkipFileCheck=true`?

Reply:
`File` is an item that only the custom packaging system understands. `None` instead is supported by both  nuget and the custom packaging system. This will help @Anipik with this pkgproj conversion.

Reply:
Ok cool.